ui/app/AppLayouts/Onboarding/views/ConfirmPasswordView.qml

```diff
@@ -21,6 +21,7 @@ Page {
     signal backClicked()
 
     anchors.fill: parent
+    background: null
```

Comment:
Could it be better to assign to `Style.current.background`?

Reply:
this is changed (and fixed) again in https://github.com/status-im/status-desktop/pull/4985, other than this I would avoid creating more Rectangles if not needed.